src/command.js

```
@@ -109,6 +110,7 @@ export default (watchPath) => {
       .option('-l, --log-level [n]', 'Log level', handelInputValues('LOG_LEVEL', watchConfig.logLevel, defaultLogLevel))
       .option('-o,--output-type [n]', 'Output type', handelInputValues('OUTPUT_TYPE', watchConfig.outputType, defaultOutputType))
       .option('-e,--access-token [n]', 'etherscan access token', handelInputValues('ACCESS_TOKEN', watchConfig.accessToken, defaultAccessToken))
+      .option('-c,--colors [n]', 'use color 1 for ansi, 2 for chalk, any other for none', handelInputValues('COLORS', watchConfig.colors, defaultColors))
```

let's make it three values 1,2,empty any other value should raise an exception